certified-connectors/bitskout/apiDefinition.swagger.json

```diff
@@ -163,7 +163,737 @@
+										"RawJSON": {
+											"type": "string",
+											"description": "RawJSON"
+										},
+										"qrcode": {
+											"type": "string",
+											"description": "qrcode"
+										}
+									},
+									"description": "outputs"
+								}
+							}
+						}
+					}
+				},
+				"summary": "Extract QR Code from Document",
+				"description": "Extract QR code value from provisioned documents using Bitskout predefined plugin",
+				"operationId": "Extract_QR_From_Document",
+				"parameters": [{
+					"name": "body",
+					"in": "body",
+					"required": false,
+					"schema": {
+						"type": "object",
+						"properties": {
+							"file_url": {
+								"type": "string",
+								"description": "file_url"
+							}
+						}
+					}
+				}]
+			}
+		},
+		"/actions/barcodes": {
+			"post": {
+				"responses": {
+					"200": {
+						"description": "default",
+						"schema": {
+							"type": "object",
+							"properties": {
+								"outputs": {
+									"type": "object",
+									"properties": {
+										"RawJSON": {
+											"type": "string",
+											"description": "RawJSON"
+										},
+										"barcode": {
+											"type": "string",
+											"description": "barcode"
+										}
+									},
+									"description": "outputs"
+								}
+							}
+						}
+					}
+				},
+				"summary": "Extract Barcode from File",
+				"description": "Extract Barcode value from provisioned file using Bitskout predefined plugin",
+				"operationId": "Extract_Barcode_From_File",
+				"parameters": [{
+					"name": "body",
+					"in": "body",
+					"required": false,
+					"schema": {
+						"type": "object",
+						"properties": {
+							"file_url": {
+								"type": "string",
+								"description": "file_url"
+							}
+						}
+					}
+				}]
+			}
+		},
+		"/actions/cold_response": {
+			"post": {
+				"responses": {
+					"200": {
+						"description": "default",
+						"schema": {
+							"type": "object",
+							"properties": {
+								"outputs": {
+									"type": "object",
+									"properties": {
+										"RawJSON": {
+											"type": "string",
+											"description": "RawJSON"
+										},
+										"Result": {
+											"type": "string",
+											"description": "Analysis result - Interested or not Interested",
+											"title": "Result"
+										}
+									},
+									"description": "outputs"
+								}
+							}
+						}
+					}
+				},
+				"summary": "Detect Response to Cold Email",
+				"description": "Detects response to Cold Email using predefined Bitskout plugin",
+				"operationId": "Detect_Response_Cold_Email",
+				"parameters": [{
+					"name": "body",
+					"in": "body",
+					"required": false,
+					"schema": {
+						"type": "object",
+						"properties": {
+							"text": {
+								"type": "string",
+								"description": "text"
+							}
+						}
+					}
+				}]
+			}
+		},
+		"/actions/haro": {
+			"post": {
+				"responses": {
+					"default": {
```

Comment:
@saferoombiz it is not allowed to have a default response only. please define a proper response. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses). 
And please note that the 'default' response should not have schema definition. Schemas should be defined on expected responses only.